x-pack/test/search_sessions_integration/tests/apps/dashboard/async_search/sessions_in_space.ts

```diff
@@ -18,17 +19,28 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     'visChart',
     'security',
     'timePicker',
+    'searchSessionsManagement',
   ]);
   const dashboardPanelActions = getService('dashboardPanelActions');
   const browser = getService('browser');
   const searchSessions = getService('searchSessions');
+  const kibanaServer = getService('kibanaServer');
+  const toasts = getService('toasts');
 
-  // Failing: See https://github.com/elastic/kibana/issues/112732
-  describe.skip('dashboard in space', () => {
```

Comment:
Test was skipped because of https://github.com/elastic/kibana/issues/113520
Adding a workaround for flakiness and also adjusting a test to go throng mgmt page so that it goes through URL service flow